ui/components/Header.js

```diff
@@ -89,6 +89,21 @@ const K8sContextConnectionChip_ = ({
     ctx.connection_id,
   );
 
+  const getDefaultIcon = (kind) => {
```

Comment:
These need to be retrieved from each respective Connection Definition.

We reviewed what a Definition is on yesterday's Meshery Dev meeting. See YouTube channel for the recording.

Reply:
@KUSHAL-BANKHEDE have you gone through the connection definition? There should be svg value over there. Checkout out dashboard tabs icons, they are been done in same way.